app/components/FileUploader.tsx

```
@@ -1,53 +1,55 @@
+                                        {selectedFile.name}
                                     </p>
                                     <p className="text-sm text-gray-500">
-                                        {formatSize(file.size)}
+                                        {formatSize(selectedFile.size)}
                                     </p>
                                 </div>
                             </div>
-                            <button className="p-2 cursor-pointer" onClick={(e) => {
-                                onFileSelect?.(null)
-                            }}>
+                            <button className="p-2 cursor-pointer" onClick={handleRemoveFile}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)